resources/extra-large-conference/pre-configure.sh

```diff
@@ -23,12 +25,14 @@ do
     ln -s /etc/prosody/certs /etc/prosody-v${i}/certs
     cp prosody.cfg.lua.visitor.template /etc/prosody-v${i}/prosody.cfg.lua
     sed -i "s/vX/v${i}/g" /etc/prosody-v${i}/prosody.cfg.lua
+    sed -i "s/jitmeet.example.com/$JICOFO_HOSTNAME/g" /etc/prosody-v${i}/prosody.cfg.lua
```

Comment:
Is it worth using some sort of templating instead of `sed`? Now I see that `jitmeet.example.com` is a template string rather than a 10 year old reference :)